Lib/test/libregrtest/refleak.py

```diff
@@ -5,6 +5,7 @@
 import warnings
 from inspect import isabstract
 from test import support
+from _abc import _get_dump
```

Comment:
This import can fail.

Reply:
But isn' `refleak.py` supposed to be used only by CPython (and no other implementation)? Or you mean there may be custom CPython builds that prohibit the C version?

Reply:
> Or you mean there may be custom CPython builds that prohibit the C version?

Yeah, I had that use-case in mind. Don't know if there are such environments, but just in case I think it's better to always have a pure Python fallback.